style.css

```diff
@@ -599,6 +609,119 @@ footer .e_mail {
   padding-bottom: 10px;
 }
 
+.popop-buttons-desk {
+  display: none;
+}
+
+.popop {
+  background-color: #c1c7d0;
+  padding: 16px 20px;
+  z-index: 20;
+}
+
+.popop-container {
+  background-color: white;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.closeTab {
+  border-radius: 8px;
+  font-size: 40px;
+  color: black;
+  position: relative;
+  top: 0;
+  left: 95%;
+}
```

Comment:
- [ ] Kindly fix the color and size of the close icon in the details popup window, so that it resembles the Figma template as much as possible.  Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/180025960-6e153ae9-ecd2-4fee-bbf0-22d51dd7433a.png)  |  ![image](https://user-images.githubusercontent.com/60618877/180025685-a3be67e5-94b5-4bd6-948f-322e1889901d.png)